third_party/3.6/click/decorators.pyi

```diff
@@ -57,6 +57,8 @@ def group(
     short_help: str = None,
     options_metavar: str = '[OPTIONS]',
     add_help_option: bool = True,
+    # User-defined
+    **kwargs,
```

Comment:
Fixed.  I assumed `Any` would be the default interpretation.  Or maybe this is a convention, to be explicit?